docs/yorm.md

```diff
@@ -3,10 +3,10 @@
 ## Mini-ORM für YForm 
 (ORM = Object-relational mapping = Objektrelationale Abbildung)
 
-Yorm erlecihtert den Umgang mit in YForm Table Manager angemeldeten Tabellen und deren Daten. So ist es möglich mittels eigener Modelclasses die Daten zu verarbeiten und aufbereitet auszugeben. Werden im Table Manager neue Felder hinzugefügt oder entfernt, passen sich über Yorm ausgegegebene Formulare sofort darauf an. Die übliche PIPE oder PHP-Programmierung entfällt. Formulare müssen meist nur durch wenige Parameter ergänzt werden um sofort zu funktionieren.  
+Yorm erleichtert den Umgang mit in YForm Table Manager angemeldeten Tabellen und deren Daten. So ist es möglich mittels eigener Modelclasses die Daten zu verarbeiten und aufbereitet auszugeben. Werden im Table Manager neue Felder hinzugefügt oder entfernt, passen sich über Yorm ausgegegebene Formulare sofort darauf an. Die übliche PIPE oder PHP-Programmierung entfällt. Formulare müssen meist nur durch wenige Parameter ergänzt werden um sofort zu funktionieren.  
 
-> - [YOrm ohne eigene Model Class verwenden](#ohne-model-class)
-> - [YOrm mit eigener Model Class verwenden](#eigene-model-class)
```

Comment:
Ich würde YOrm in den erklärungen so beibehalten, das gleiche wie bei YForm zbm.Weil ich finde so sieht es jedenfalls in einem Text besser aus, wenn man YOrm anstatt Yorm schreibt

Reply:
@gharlan wie ist es von deiner Seite aus gedacht? Ich finde auch, dass YOrm besser auf den ersten Blick zu YForm unterscheidbar ist als Yorm

Reply:
Ansonsten habe ich bisher nichts gefunden, was man verbessern könnte :blush:.

Reply:
Von mir war es mal als YOrm gedacht, würde das also präferieren.

Reply:
Yorm sollte es damals heißen. Geschrieben wie gesprochen. Nicht Y-Orm (Wai-Orm oder Wai-O-ER-Em).

Reply:
Ich schreibe es YOrm und spreche es trotzdem yorm aus.